designer/client/src/containers/Notifications.tsx

```diff
@@ -44,9 +44,15 @@ const prepareNotification =
     };
 
 const handleRefresh =
-    ({ scenarioName, toRefresh }: BackendNotification, currentScenarioName: string, processVersionId: number): ThunkAction =>
+    (
+        { scenarioName, toRefresh }: BackendNotification,
+        currentScenarioName: string,
+        processVersionId: number,
+        currentProcessingType: string,
+        currentIsFragment: boolean,
+    ): ThunkAction =>
     (dispatch) => {
-        if (!scenarioName || scenarioName !== currentScenarioName) {
+        if (scenarioName && scenarioName !== currentScenarioName) {
```

Comment:
This new condition causes a problem outside view of scenario. When you load scenario list view, the notifications also fire and reloading configuration will result in a call under `api/processDefinitionData/undefined?isFragment=undefined` . I think need different conditions for refreshing creator vs state and activity since for creator we have no scenario name.

Reply:
Fixed